docs/operating-eck/troubleshooting/common-problems.asciidoc

```diff
@@ -260,3 +260,69 @@ If you accidentally upgrade one of your Elasticsearch clusters to a version that
 The reason for this validation is that ECK will not allow downgrades as this is not supported by Elasticsearch and once the data directory of Elasticsearch has been upgraded there is no way back to the old version without a link:https://www.elastic.co/guide/en/elasticsearch/reference/current/setup-upgrade.html[snapshot restore].
 
 These two upgrading scenarios, however, are exceptions because Elasticsearch never started up successfully. If you annotate the Elasticsearch resource with `eck.k8s.elastic.co/disable-downgrade-validation=true` ECK allows you to go back to the old version at your own risk. If you also attempted an upgrade of other related Elastic Stack applications at the same time you can use the same annotation to go back. Remove the annotation afterwards to prevent accidental downgrades and reduced availability.
+
+[id="{p}-{page_id}-815-reconfigure-role-mappings"]
+== Manual steps to do after upgrading to 8.15.3 from 8.14.x or 8.15.x to reconfigure operator role mappings
+
+You have operator role mappings defined in a StackConfigPolicy, and you upgraded from 8.14.x or 8.15.x, to 8.15.3.
```

Comment:
```suggestion
You have role mappings defined in a StackConfigPolicy, and you upgraded from 8.14.x or 8.15.x, to 8.15.3.
```
we don't use the `operator` concept (which Elasticsearch uses)  anywhere else in the docs and it might be confusing given that the term is overloaded on Kubernetes.

Reply:
🤔 you marked this as resolved but i don't see any change?

Reply:
1ac2efd8f1de

Reply:
Hm, there were two places with operator. 
cd5a5b2